src/components/tiptap/notificationWidget/NotificationWidget.tsx

```diff
@@ -56,7 +56,7 @@ export const NotificationWidget = () => {
           style={{ position: 'relative', cursor: 'pointer' }}
         >
           <Typography variant='h2' datatype='draggable-item'>
-            You have {taskCount} tasks left to complete
+            You have {taskCount} task/s left to complete
```

Comment:
Please implement some pluralization util so it can be singular or plural conditionally
```
You have 1 task left to complete
```
```
You have 9 tasks left to complete
```

Reply:
good catch. next commit adds the condition